Tests/TimeKeeperTest.cs

```diff
@@ -0,0 +1,42 @@
+using System.Timers;
+using HotelPriceScout.Data.Function;
+using Xunit;
+
+namespace Tests
+{
+    public class TimeKeeperTest
+    {
+        [Fact]
+        public void TimeKeeperMinuteTriggerConstructorCreatesTimerTest()
+        {
+            //Arrange
+            int minutes = 10;
+            void Receiver(object obj, ElapsedEventArgs args)
+            {
+            };
+            
+            //Act
+            ITimeKeeper timeKeeper = new TimeKeeper(minutes, Receiver);
+            
+            //Assert
+            Assert.IsType<Timer>(timeKeeper.Timer);
+        }
+        
+        [Fact]
+        public void TimeKeeperDailyTriggerConstructorCreatesTimerTest()
+        {
+            //Arrange
+            int hourOfDay = 10;
+            int minuteOfDay = 10;
+            void Receiver(object obj, ElapsedEventArgs args)
+            {
+            };
+            
+            //Act
+            ITimeKeeper timeKeeper = new TimeKeeper(hourOfDay, minuteOfDay, Receiver);
+            
+            //Assert
+            Assert.IsType<Timer>(timeKeeper.Timer);
+        }
+    }
+}
```

Comment:
No newline at end of file 
![st,small,845x845-pad,1000x1000,f8f8f8](https://user-images.githubusercontent.com/72670627/143415432-63059fb3-e54a-4a37-8e30-248853272c61.jpg)